src/scroll-timeline-base.js

```diff
@@ -587,115 +505,22 @@ export class ViewTimeline extends ScrollTimeline {
         return "inactive";
```

Comment:
I think you should be able to return super.phase to invoke the base class getter and not need to repeat the logic.

Reply:
Agreed. Written prior to removing the obsolete code from  ScrollTimeline.  I'll refactor.